ONSdigital/response-operations-ui
# UI for adding and updating remaining optional dates

# Motivation and Context

Before there was no way to add and update reference period start,
reference period end and employment dates. This PR adds this ability
to the UI. Also broken down the monster collection-exercise.html file
into small slightly more manageable files.

# What has changed



collection-exercise.html broken to smaller files by section
Added ability to add and update reference period start, reference period end and employment dates.

# How to test?



Ran ATs 
Manually try adding dates and updating dates.
Fixed guidance messages for both creating and updating all dates
Fixed error messages for both creating and updating all dates

Might see bugs if using master branch collection exercise service use >> https://github.com/ONSdigital/rm-collection-exercise-service/pull/115

Out of scope for this PR:
- Adding, event is in the future validation

# Links



https://github.com/ONSdigital/rm-collection-exercise-service/pull/115

![image](https://user-images.githubusercontent.com/22023234/44521084-c9424180-a6c9-11e8-86ed-5dc6db1e4b2d.png)
